frontend/src/Commons/Input/Field/CustomField.style.ts

```diff
@@ -13,7 +12,7 @@ type StyledFieldProps = {
 const StyledField = styled.div<StyledFieldProps>`
   display: flex;
   flex-direction: column;
-  width: 100%;
+  width: ${(props) => props.width};
```

Comment:
:ok_hand: 
I think this follows the mockups with more fidelity, although I personally prefer to have the CustomFields 100% on mobile, makes it easier in my opinion. I'll take it out on @Shakespeares-spy though. Good job!